Added two separate configs

warmachinesocial opened the pull request:
Have added 2 xcschemes and xcconfigs file dev and prod such that makes file easier on env variables later down the road. Let me know if you're fine with it

M3DZIK commented:
@aeoliux

M3DZIK commented:
@warmachinesocial could you fix the build in github actions?

warmachinesocial commented:
@M3DZIK sorry forgot that my xcode automatically signs off, would just turn it off

aeoliux commented:
1. GitHub Action fails
2. What is the purpose of custom environment? App works with custom API servers. You just need to specify one while creating account and logging in. Also no information about ${ROOT_URL} in Info.plist. There should be a default URL api.librepass.org .
3. What is the purpose of xcschemes? Unsigned IPA is used for distribution. To test app on emulator you just need to run it and to test it on real device connected via USB cable choose development team and install it via Xcode.

warmachinesocial commented:
1. There are no env variables at this stage, however setting up from now to make life easier. In the case of following git flow with 3 branches(dev,staging,prod) we can just select specific xcsheme to automatically take the env variables from Info.plist. 
2. Hard coded values not good
3. Well I had my xcode by default on automatic signing with my ID therefore it threw the above error. 
4. Yes the build fails because I have removed(renamed) the old xcscheme and added new one also :- (LibrePass(Dev) and LibrePass(Prod)).
5. Do we have a staging environment? Based on the code I thin we just have 2 of them, correct me if I'm wrong

aeoliux commented:
We have only one branch and we don't need more

warmachinesocial commented:
No worries
